### PR TITLE
[RHCLOUD-20542] Adding (cron)job watch-controlled list

### PR DESCRIPTION
**Pull request by gburges**

### CODE REVIEW

**Comment by psav** [comment]
Need to rebase and run `make pre-push`

**Comment by gburges** [comment]
> Need to rebase and run `make pre-push`

did it, but the check still fails, tried make build-template and recommit too. Any ideas?

**Comment by psav** [comment]
`go mod tidy`

**Review by psav — CHANGES_REQUESTED** [review]
Sorry @gburges a few things here
1) the subscriptions yaml shouldn't be included here in this PR - let's do it in a separate one
2) Where did all the changes to the go.sum come from? try `go mod tidy -compat=1.17`

**Comment by gburges** [comment]
> Sorry @gburges a few things here
> 
>     1. the subscriptions yaml shouldn't be included here in this PR - let's do it in a separate one
>  removed
>     2. Where did all the changes to the go.sum come from? try `go mod tidy -compat=1.17`
> yeah, might have overupdated my local setup - now it's ok

File: controllers/cloud.redhat.com/clowdapp_controller.go [hunk 1]
```diff
@@ -54,6 +55,7 @@ import (
 	_ "github.com/RedHatInsights/clowder/controllers/cloud.redhat.com/providers/featureflags"
 	_ "github.com/RedHatInsights/clowder/controllers/cloud.redhat.com/providers/inmemorydb"
 	_ "github.com/RedHatInsights/clowder/controllers/cloud.redhat.com/providers/iqe"
+	_ "github.com/RedHatInsights/clowder/controllers/cloud.redhat.com/providers/job"
```
Comment (psav) [inline controllers/cloud.redhat.com/clowdapp_controller.go:58]: Remove this line

**Comment by psav** [comment]
This actually isn't required - we found a different bug which is causing the behaviour you described and this won't fix it :)

**Pull request closed without merging**